symbulate/results.py

```diff
@@ -286,22 +287,46 @@ class RVResults(Results):
 
     def plot(self, type=None, alpha=None, normalize=True, jitter=False, 
         bins=30, **kwargs):
+        if type is None:
```

Comment:
What's the point of this block of code? It doesn't seem to do anything.

Reply:
I have this there since I'm handling the default type situations after going into the different number of variable cases. Should I be handling this differently?

Reply:
I see what you mean here now. Disregard above.

Reply:
Oh I see what you're doing. I think it would be clearer to do:

```
if type is not None:
  if isinstance(type, str):
    ...
  elif isinstance(type, (tuple, list)):
    ...